appveyor.yml

```diff
@@ -46,7 +49,6 @@ test_script:
   - bundler --version
   - bash ./script/cibuild
 
-cache:
+#cache:
   # If one of the files after the right arrow changes, cache will be skipped
-  - C:\Ruby%RUBY_FOLDER_VER%\bin -> Gemfile,jekyll.gemspec,appveyor.yml
-  - C:\Ruby%RUBY_FOLDER_VER%\lib\ruby\gems\%GEMS_FOLDER_VER% -> Gemfile,jekyll.gemspec,appveyor.yml
+#  - 'vendor -> appveyor.yml,Gemfile,jekyll.gemspec'
```

Comment:
Haha yeah caching the system-wide gems folder is not ideal. You shouldn't need `--deployment` for us (that's for running `bundle install` before you boot a server and it's why heroku, for example, requires the `Gemfile.lock`) – can we just do a custom list of `--without` to make sure we only install what we need?